src/Console/Commands/FindMissingTranslationStrings.php

```diff
@@ -131,21 +132,31 @@ protected function printErrors(array $errors, OutputInterface $output): void
      * @return \Illuminate\Support\Collection
      */
     protected function findInArray(string $baseLocale, mixed $locale)
+
     {
         if ($baseLocale === $locale) {
             return Collection::empty();
         }
-        return Collection::make($this->files->files(lang_path($baseLocale)))
-            ->mapWithKeys(function (SplFileInfo $file) {
-                return [$file->getFilenameWithoutExtension() => $this->translator->get($file->getFilenameWithoutExtension())];
-            })
-            ->dot()
-            ->keys()
+
+        $data = Collection::make($this->translator->get('*'));
```

Comment:
Can you let me know what this part does?

Reply:
Yeah. `$this->translator->get('*')` loads the global (non-namespaced) translations in `lang/{$baseLocale}.json`.

I wonder if it might be better to use the found translations via scanning to build a list of groups/namespaces to load, that would support non-standard loaders.

Reply:
Yeah, I don't think there's anything technically preventing you from using a json file for the base locale, but it doesn't seem like it is very common.